Makefile

```diff
@@ -45,10 +45,10 @@ BUNDLE_DEFAULT_CHANNEL := --default-channel=$(DEFAULT_CHANNEL)
 endif
 BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 
-QUAY_USERNAME ?=
-QUAY_PASSWORD ?=
+ICR_USERNAME ?=
+ICR_PASSWORD ?=
 
-MARKDOWN_LINT_WHITELIST=https://quay.io/cnr
+MARKDOWN_LINT_WHITELIST=https://icr.io/cnr
```

Comment:
I'm not sure what this lint whitelist is, so I don't know if it actually needs to change or not